distance_validation.py

```diff
@@ -0,0 +1,70 @@
+# Created by Tedi Mitiku 07/07/20
+# This script is for verifying our euclidean distance assumption on
+# the 7D user attribute vector space.
+
+import math
+import random
+import numpy as np
+import pandas as pd
+import normalizing_data
+
+#Take in user attribute data as DataFrame
```

Comment:
Typically, if you ran the linter it would say put a space before the start of comments, like above.